File: python/dish-decision.py
import json
import urllib.request

key = input('Your public key: ')

season = input('What Season is it? (Spring, Summer, Fall, Winter)  ')
guests = input('How many Guests?  ')

inputs = json.dumps({
    'season': {'value': season,
               'type': 'string'},
    'guests': {'value': int(guests),
               'type': 'integer'}
}).encode('utf-8')

request = urllib.request.Request(
    'https://dmn.lab.camunda.com/api/v1/decision/{}'.format(key),
    data=inputs,
    headers={'Content-Type': 'application/json'})

with urllib.request.urlopen(request) as response:
    body = response.read().decode('utf-8')

result = json.loads(body)['outputs']['dish']['values'][0]
print('You should have ' + result)

File: php/dish-decision.php
<?php
	$data = array(
		'season' => array(
			'type' => 'string',
		    'value' =>  $_GET["season"]
		),
		'guests' => array(
			'type' => 'integer',
		    'value' =>  intval($_GET["guestCount"])
		)
	);

	$service_url = 'https://dmn.camunda.cloud/api/v1/decision/example-dish';

	$curl = curl_init($service_url);

	curl_setopt($curl, CURLOPT_HTTPHEADER, array('Content-Type: application/json'));
	curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
	curl_setopt($curl, CURLOPT_POST, true);
  	curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, false);

	$data_string = json_encode($data);

	curl_setopt($curl, CURLOPT_POSTFIELDS, $data_string);

	$curl_response = curl_exec($curl);

	curl_close($curl);

	$decoded = json_decode($curl_response);

    echo "You should have " . $decoded->outputs->dish->values[0];
?>
